standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/common/StatsSetupConst.java

```diff
@@ -79,6 +80,21 @@ public String getAggregator(Configuration conf) {
     public abstract String getAggregator(Configuration conf);
   }
 
+  public enum IcebergStatsSource {
+    ICEBERG,
+    METASTORE;
+
+    public static IcebergStatsSource fromString(String value) throws MetaException {
```

Comment:
would be better to define lookup map for this, see OperationType

Reply:
Done